src/pages/resultsView/enrichments/MiniFrequencyScatterChart.tsx

```diff
@@ -25,6 +25,69 @@ export interface IMiniFrequencyScatterChartProps {
     onGeneNameClick: (hugoGeneSymbol: string, entrezGeneId: number) => void;
     onSelection: (hugoGeneSymbols: string[]) => void;
     onSelectionCleared: () => void;
+    selectedGenesSet:{[hugoGeneSymbol:string]:any};
+}
+
+interface IPointProps {
```

Comment:
I know this is repetitive with the other scatter chart, but Im not sure whether it actually makes sense to generalize the code rather than copy paste and modify it

@alisman @kalletlak

Reply:
I prefer to generalize here as more than 90% of it is the same